packages/labs/virtualizer/src/Virtualizer.ts

```diff
@@ -720,17 +718,14 @@ export class Virtualizer {
         if (child) {
           child.style.position = 'absolute';
           child.style.boxSizing = 'border-box';
-          child.style.transform = `translate(${left}px, ${top}px)`;
           if (width !== undefined) {
             child.style.width = width + 'px';
           }
           if (height !== undefined) {
             child.style.height = height + 'px';
           }
-          (child.style.left as string | null) =
-            xOffset === undefined ? null : xOffset + 'px';
-          (child.style.top as string | null) =
-            yOffset === undefined ? null : yOffset + 'px';
+          child.style.left = `${(xOffset === undefined ? 0 : xOffset) + left}px`;
+          child.style.top = `${(yOffset === undefined ? 0 : yOffset) + top}px`;
```

Comment:
In my context I don't want a new stacking context being created, this is an issue. Avoid using transform can fix that.
In addition I had to put `style="contain: size"` to the scrolling container, to avoid `layout` value https://github.com/lit/lit/blob/05691ba4848f1bcbec06df7dcd63ee024589ed47/packages/labs/virtualizer/src/Virtualizer.ts#L362